bin/repair/delete_invisible_lanes_and_their_descendants

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+"""Delete invisible lanes associated with one or more libraries."""
+import os
+import sys
+
+from core.scripts import DeleteInvisibleLanesScript
+
+bin_dir = os.path.split(__file__)[0]
+package_dir = os.path.join(bin_dir, "..", "..")
+sys.path.append(os.path.abspath(package_dir))
```

Comment:
I know this is already merged, but it doesn't look like it can work as is because we have the import statement (l. 6) before we fix up the search path (ll. 8-10). We probably need to rewrite something like the following for it to work properly:
```
...
bin_dir = os.path.split(__file__)[0]
package_dir = os.path.join(bin_dir, "..", "..")
sys.path.append(os.path.abspath(package_dir))

from core.scripts import DeleteInvisibleLanesScript
...
```

Reply:
PR forthcoming to address.